azure/converters/extendedlocation_test.go

```diff
@@ -63,17 +62,17 @@ func TestExtendedLocationToNetworkSDK(t *testing.T) {
 	tests := []struct {
 		name string
 		args *infrav1.ExtendedLocationSpec
-		want *network.ExtendedLocation
+		want *armnetwork.ExtendedLocation
```

Comment:
I don't need to update this test either in this PR. Will follow up in another PR.